### DISCUSSION TITLE
saving  and opening several tab in the comfyUI   for txt2img , img2img and else

### DISCUSSION THREAD

**Discussion by mostafablender** [Comfy-Org/ComfyUI#2065] [Ideas]
Hi  . i really  dont like the way comfyUI is right now  every time i have to put a jason file or drag or drop  something in it  i like if it be possible to have several window for anything i made and i can load them inside of the comfyUi itself  this way changing to txt2img or using loras or controlnet and txt2video going to be faster for me ... 
my idea is to adding several small tab on up of the Ui each one is a node you can select + and make new one and they will be recorded inside of the comfyUI so everytime you open your comfyUI  you see all of them

**Comment by remybonnav** [comment]
You can probably create al lyour workflows in same view. Group each and just bypass the ones that you don't need (right click, bypass node).

**Reply by jn-jairo** [reply]
That is how I do, I have a big workflow with groups that I bypass what I don't need.
